src/main/java/uk/co/ramp/covid/simulation/lockdown/FullLockdownComponent.java

```diff
@@ -0,0 +1,43 @@
+package uk.co.ramp.covid.simulation.lockdown;
+
+import uk.co.ramp.covid.simulation.place.CommunalPlace;
+import uk.co.ramp.covid.simulation.population.Person;
+import uk.co.ramp.covid.simulation.population.Population;
+import uk.co.ramp.covid.simulation.util.Time;
+
+public class FullLockdownComponent extends LockdownComponent {
+    
+    private double sDist;
+
+    public FullLockdownComponent(Time s, Time e, Population p, double sDist) {
+        super(s, e, p);
+        this.sDist = sDist;
+    }
+
+    @Override
+    protected void start() {
```

Comment:
This component seems to handle two different aspects of lockdown - places entering lockdown and people being furloughed.  Would it make sense for these to be in separate components?

Reply:
I agree that in the future this will be two components - this was mostly as an example of getting the "original" behaviour.